src/actions/accountCreator.js

```diff
@@ -60,3 +60,64 @@ export function startFriendbotRequest(target) {
       })
   }
 }
+
+export const GENERATE_MUXED_ACCOUNT = 'GENERATE_MUXED_ACCOUNT';
+export function generateMuxedAccount(publicKey, muxedAccountId) {
+  try {
+    const muxedAccount = new MuxedAccount(new Account(publicKey, "0"), muxedAccountId);
```

Comment:
We won't be doing any transactions with this account, using `0` sequence number for simplicity.